apps/ledger-live-desktop/src/renderer/Default.jsx

```diff
@@ -192,7 +192,16 @@ export default function Default() {
             <Switch>
               <Route path="/onboarding" render={props => <Onboarding {...props} />} />
               <Route path="/sync-onboarding" render={props => <SyncOnboarding {...props} />} />
-              <Route path="/post-onboarding" render={() => <PostOnboardingScreen />} />
+              <Route
+                path="/post-onboarding"
+                render={() => (
+                  <>
+                    <PostOnboardingScreen />
+                    <Drawer />
```

Comment:
what's the point of this?

Reply:
`<Drawer />` was a children of `/` (line 211) so it would not render in the post onboarding. I needed to pass it as a children to the post onboarding to be render.